cost/azure/databricks/rightsize_compute/README.md

```diff
@@ -0,0 +1,126 @@
+
+##### Grant Permission to Azure Service Principal in all Azure Databricks Workspaces
+
+1. Get Service Principal's Client ID
+
+   You can get this from the Azure Portal or via Flexera > Automation > Credentials and get the Client ID for the Azure RM Credential that is being used.  The Azure SP that is used for other Flexera Azure Policy Templates can be used for the Databricks Policy Templates.
+
+1. Add Service Principal to all DB Workspace using the Client ID
+
+   Databricks Workspace  Admin Settings > Service Principal (i.e. `https://{workspaceUrl}/?#setting/accounts/servicePrincipals` )
+
+1. Grant Service Principal Permissions in DB Workspace
+
+   We currently recommend adding Service Principal to `admin` group so it can see all clusters and compute resources within the cluster.
+
+   Databricks Workspace > Admin Settings > Groups (i.e. `https://{workspaceUrl}/?#setting/accounts/groups` )
+
+##### Create `OAuth2` Credential in Flexera
+
+Replace these:
+
+- `{access_token}`
+- `{flexeraProjectId}`
+- `{Credential Name}`
+- `{credentialId}`
+- `{clientId}`
+- `{clientSecret}`
+- `{tenantId}`
+
+```sh
+curl 'https://api.flexera.com/cred/v2/projects/{flexeraProjectId}/credentials/oauth2/{credentialId}' -X PUT -H 'Content-Type: application/json' -H 'Api-Version: 1.0' -H 'Authorization: Bearer {access_token}' --data-raw '{"description":"","name":"{Credential Name}","grantType":"client_credentials","tags":[{"key":"provider","value":"databricks"}],"clientCredentialsParams":{"additionalParams":{"resource":"2ff814a6-3304-4ab8-85cb-cd0e6f879c1d"},"clientId":"{clientId}","clientSecret":"{clientSecret}"},"tokenUrl":"https://login.windows.net/{tenantId}/oauth2/token"}'
+```
+
+#### Option 2: Authenticate to Databricks using Databricks Personal Access Token
+
+##### Create `API Key` Credential in Flexera
+
+Replace these:
+
+- `{access_token}`
+- `{flexeraProjectId}`
+- `{Credential Name}`
+- `{credentialId}`
+- `{personalAccessToken}`
+
+```sh
+curl 'https://api.flexera.com/cred/v2/projects/{flexeraProjectId}/credentials/oauth2/{credentialId}' -X PUT -H 'Content-Type: application/json' -H 'Api-Version: 1.0' -H 'Authorization: Bearer {access_token}' --data-raw '{"description":"","name":"{Credential Name}","field":"Authorization","location":"header","type":"Bearer","tags":[{"key":"provider","value":"databricks"}],"key":"{personalAccessToken}"}'
+```
+
+***Important*** - When creating an Applied Policy using a Databricks Personal Access Token credential, you must provide just 1 workspace identifier (Name or ID) in `param_databricks_workspace_list`.  Personal Access Tokens are workspace-scoped and a single applied policy cannot traverse multiple Databricks workspaces with a single Personal Access Token.
+
+## Supported Clouds
+
+- Azure
+
+## Cost
+
+This Policy Template does not incur any cloud costs
+
+## Other Notes
+
+### Results where `Databricks Cluster ID` == `null`
```

Comment:
Might be worth rephrasing this in a way that would be less obtuse to non-programmers. Normies don't know what == means.

Reply:
Changed `==` to `equals` in 851728c20c2a7cc5ca94dac013c36e98a3043cad